<!DOCTYPE html>
<html lang="en">
    <div id="medium_blog_container">
        <!-- Here we will mount our component -->
    </div>
    <body>
        <script src="https://unpkg.com/react@18/umd/react.development.js" crossorigin></script>
        <script src="https://unpkg.com/react-dom@18/umd/react-dom.development.js" crossorigin></script>
    </body> 

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Data Manipulation Portfolio</title>
    <style>
        /* Add your custom CSS styles here */
        body {
            font-family: Arial, sans-serif;
            line-height: 1.6;
            margin: 0;
           padding: 0;
            padding-left: 20px;
            background-image: url('picture3.jpg') ;/*('pagePic.jpg')*/
            background-size: cover;
            
        }
        #container {
        display: flex;
        flex-direction: column;
        align-items: center;
        justify-content: center;
        min-height: 100vh;
        margin-left: 20px;
        text-align: center;
    }

    /* Additional CSS for your content */
    /* For example, you can style the header text */
    header {
        font-size: 20px;
    }
    p{
        /*color:rgb(0, 26, 255)*/
        color: black /*whitesmoke*/

    }
    h1{
       color:black /*red*/
    }

    h2{
        color:blanchedalmond/*red*/

    }
    /* ... Additional CSS styling ... */
    </style>
</head>

<body>
    <header>
        <h1>Data Manipulation Portfolio</h1>
        <!-- Add your personal information like name, email, and LinkedIn profile here -->
        <address>
            Email: ambrolum@gmail.com<br>
            GitHub: <a href="https://github.com/Ambro19">github.com/Ambro19</a>
    </address>
    </header>

    <section>
        <h2>Select Projects</h2>
        <!-- Add a brief description of your selected LabVIEW projects -->
        <p>Human biology can be complex, in part due to the function and interplay of the body's 
            approximately 37 trillion cells, which are organized into tissues, organs, and systems. 
            However, recent advances in single-cell technologies have provided unparalleled insight 
            into the function of cells and tissues at the level of DNA, RNA, and proteins. 
            Yet leveraging single-cell methods to develop medicines requires mapping causal links 
            between chemical perturbations and the downstream impact on cell state. These experiments 
            are costly and labor intensive, and not all cells and tissues are amenable to high-throughput 
            transcriptomic screening. If data science could help accurately predict chemical perturbations 
            in new cell types, it could accelerate and expand the development of new medicines.</p>
    </section>

    <section>
        <h2>Project Descriptions</h2>
        <!-- Write detailed descriptions for each project -->
            <p> Project 1: Kaggle Machine Learning Competition: Open-Problem: Single - Cell Perturbations<br>
                Goals:
                <ul>
                    <li>The goal of this competition is to predict how small molecules change gene expression in different cell types.

                        The contribution will help develop methods to predict how cells respond to small molecule 
                        drug perturbations, which could have important applications in drug discovery and basic biology.</li>
                </ul>

                Challenges:
                <ul>
                    <li>The competition participants are asked to build models that can predict the differential expression 
                        of all 18,211 genes in the transcription dataset. In other words, they are expected to estimate how 
                        the expression of each gene changes in response to experimental treatments or compounds.</li>
                </ul>

                Solutions:
                <ul>
                    <li>To provide solution, the competition participants are tasked with modeling differential 
                        expression (DE) and how the impact of experimental perturbations on the expression levels of all 
                        genes in the dataset is estimated.</li>
                </ul>
            </p>
        <!-- Add more project descriptions as needed -->
    </section>

    <section>
        <h2>Code Samples</h2>
        <!-- Share snippets of your LabVIEW code -->
        <pre>
            <code>
             # In progress
            </code>
        </pre>
    </section>

    <!-- Add more sections for Screenshots and Media, Documentation, Learning Journey, Collaborative Projects,
         Personal Projects, Online Presence, Presentation, and Keep It Updated. -->

    <footer>
        <p>&copy; 2023 Ambroise Ngayinoko. All rights reserved.</p>
    </footer>
</body>

</html>